mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -117,16 +117,33 @@ view: +subscriptions {
     sql: 1;;
   }
 
-  dimension: months_retained {
+  dimension: months_active {
     type: number
```

Comment:
Add a description that makes it apparent how this is different from `current_age_in_months`.

Reply:
```suggestion
    description: "The age in months of the subscription when it was cancelled, or current age if still active."
    type: number
```